coderd/rbac/roles.go

```diff
@@ -70,6 +70,14 @@ func RoleMember() string {
 	return RoleName(member, "")
 }
 
+func StaticRoleOrgAdmin() string {
+	return orgAdmin
+}
+
+func StaticRoleOrgMember() string {
+	return orgMember
+}
+
```

Comment:
What's the difference between "Static" org admin and `RoleOrgAdmin()`?

Reply:
This is super annoying, but essentially I should make `RoleOrgAdmin(<org_id>)` not take an organization ID. But there is >80 usages of `RoleOrgAdmin()` and `RoleOrgMember()`. All our test APIs just take `[]string{}` for roles, but now they should take, `[]string, map[<org_id>][]string`.

That kind of refactor would be a much larger diff, and this would only serve to affect unit tests.

Reply:
I have another idea. I will rename the old ones to something else and put up a deprecated comment.

Reply:
Solution

https://github.com/coder/coder/blob/cf933079a969cda57af4d51ae0e9a4c1aff50d80/coderd/rbac/roles.go#L73-L94